### PR TITLE
Add Ethermint Testnet

### PR DESCRIPTION
**Pull request by freddyli7**
Add ethermint testnet chain

### CODE REVIEW

File: _data/chains/eip155-777.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Photon",
+    "symbol": "Photon",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.ethermint.zone",
+  "shortName": "eth",
+  "chainId": "etherminttestnet-777",
```
Comment (ligi) [inline _data/chains/eip155-777.json:16]:
```suggestion
  "chainId": 777,
```

File: _data/chains/eip155-777.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "http://54.210.246.165:8545"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Photon",
+    "symbol": "Photon",
+    "decimals": 18
+  },
+  "infoURL": "https://docs.ethermint.zone",
+  "shortName": "eth",
```
Comment (ligi) [inline _data/chains/eip155-777.json:15]:
```suggestion
  "shortName": "emint",
```
cannot be eth

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-777.json (hunk 1 above)
Comment (freddyli7) [reply _data/chains/eip155-777.json:16]: 👌

File: _data/chains/eip155-777.json (hunk 2 above)
Comment (freddyli7) [reply _data/chains/eip155-777.json:15]: 👌

**Review by ligi — APPROVED** [review]

**Pull request merged**